xarray/tests/test_dataset.py

```diff
@@ -2124,6 +2124,19 @@ def test_stack_unstack(self):
         actual = ds[['b']].stack(z=['x', 'y']).unstack('z')
         assert actual.identical(ds[['b']])
 
+    def test_stack_unstack_slow(self):
+        ds = Dataset({'a': ('x', [0, 1]),
+                      'b': (('x', 'y'), [[0, 1], [2, 3]]),
+                      'x': [0, 1],
+                      'y': ['a', 'b']})
+        stacked = ds.stack(z=['x', 'y'])
+        actual = stacked.isel(z=range(len(stacked.z))[::-1]).unstack('z')
```

Comment:
This can be simply written `range(len(stacked.z))[::-1]` -> `slice(None, None, -1)`

Reply:
Of course! thanks for the review.